src/test/scala/nl.knaw.dans.easy.multideposit/parser/MultiDepositParserSpec.scala

```diff
@@ -331,4 +331,13 @@ class MultiDepositParserSpec extends TestSupportFixture with DepositTestObjects
       case Success(_) =>
     }
   }
+
+  it should "fail if there are multiple distinct base revisions" in {
```

Comment:
nice test! However, what you're testing is `extractInstructions`, not `uuid`. The `it` at the start of this line is in the context of `uuid`, so it looks like this is part of the test suite for the `uuid` function. However, it is rather meant to be part of the test suite of `extractInstructions`. Please move the test to the correct position.

Reply:
Oeps... You already did that 😜